backend/copr_backend/background_worker_build.py

```diff
@@ -643,6 +644,14 @@ def _download_results(self):
             filter_=filter_,
         )
 
+    def _upload_results_to_storage(self):
+        """
+        Upload build results to an appropriate storage. Duplicate the data,
+        don't remove them from the original place.
```

Comment:
Can you document why we need this, namely it is weird why we split `_upload_..()` and `_cleanup()` jobs.

Reply:
In between them, we do these steps:

```
            self._check_build_success()
            # Build _succeeded_.  Do the tasks for successful run.
            failed = False
            if self.opts.do_sign:
                self._sign_built_packages()
            self._do_createrepo()
            self._parse_results()
            build_details = self._get_build_details(self.job)
            self.job.update(build_details)
            self.job.validate()
            self._add_pubkey()
```

IIRC when I tried to do the cleanup in `_upload_results_to_storage` some of them failed because they couldn't find the RPM files.

But doing `_upload_results_to_storage` and `_cleanup` in one step should be doable. Should I try?